From 915b9b18a75fb7950e99a025c21722e3f7314600 Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@nxp.com>
Date: Mon, 29 Feb 2016 18:06:53 +0800
Subject: [PATCH 0964/5242] MLK-12457 dts: imx7d-12x12-lpddr3-arm2: correct
 pwm1 pinctrl settings

commit  c518741ba78c5b95c890d91ffddf7ede72b708c6 from
https://source.codeaurora.org/external/imx/linux-imx.git

The pwm1's pin belongs to lpsr iomux. So this should
be corrected.

Signed-off-by: Fancy Fang <chen.fang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts |   14 ++++++++------
 1 file changed, 8 insertions(+), 6 deletions(-)

diff --git a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
index 8359354..b177cdb 100644
--- a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
+++ b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
@@ -684,12 +684,6 @@
 			>;
 		};
 
-		pinctrl_pwm1: pwm1grp {
-			fsl,pins = <
-				MX7D_PAD_GPIO1_IO08__PWM1_OUT 0x110b0
-			>;
-		};
-
 		pinctrl_sai1: sai1grp {
 			fsl,pins = <
 				MX7D_PAD_SAI1_MCLK__SAI1_MCLK           0x1f
@@ -841,6 +835,14 @@
 };
 
 &iomuxc_lpsr {
+	imx7d-12x12-lpddr3-arm2 {
+		pinctrl_pwm1: pwm1grp {
+			fsl,pins = <
+				MX7D_PAD_LPSR_GPIO1_IO01__PWM1_OUT	0x110b0
+			>;
+		};
+	};
+
 	imx7d-sdb {
 		pinctrl_usbotg1_vbus: usbotg1vbusgrp {
 			fsl,pins = <
-- 
1.7.9.5

